src/main/java/bob/task/TaskDecoder.java

```diff
@@ -0,0 +1,34 @@
+package bob.task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Class responsible for decoding save string to task
+ */
+public class TaskDecoder {
+    /**
+     * Method that task a save string and decode it back to task
+     * 
+     * @param taskDesc Save string of the task
+     * @return Task based on the save string
+     */
+    public static Task retTask(String taskDesc) {
```

Comment:
Should this method be renamed to more clearly contain a verb?
```suggestion
    public static Task returnTask(String taskDesc) {
```